### DISCUSSION TITLE
v1.0.0-beta.7 fails to start process

### DISCUSSION THREAD

**Discussion by ChipClark** [coreybutler/node-windows#311] [General] [answered]
I am running a NestJS/prisma API server - running prisma/client 3.12.0   
When I start the process using node-windows@1.0.0-beta.6 - the service is created and starts.

If I upgrade to node-windows@1.0.0-beta.7 - the service is created, but will not start: Error: 1067

**Comment by coreybutler** [comment, marked as answer]
beta.7 had a dependency problem. Use [beta.8.](https://github.com/coreybutler/node-windows/releases/tag/1.0.0-beta.8) instead.
